.github/workflows/checks.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches: [main, experimental]
   pull_request:
-    branches: [main, experimental]
```

Comment:
why changing this?

Reply:
This PR is pointing at your PR's branch as I wanted to do it with the refactored paths so it didn't need changing again. Therefore, had to change this to run everything in my PR

Reply:
Oh so when you remove the explicit branches, it will run for all branches?
If that's the case I'm 👍

Reply:
indeed